javascript/extractor/src/com/semmle/js/parser/RegExpParser.java

```diff
@@ -67,6 +71,8 @@ public List<Error> getErrors() {
   private List<Error> errors;
   private List<BackReference> backrefs;
   private int maxbackref;
+  private Boolean vFlagEnabled = false;
+  private Boolean uFlagEnabled = false;
```

Comment:
```suggestion
  private boolean vFlagEnabled = false;
  private boolean uFlagEnabled = false;
```

`Boolean` is an object wrapping a primitive `boolean` value, not what we need.